images/linux/scripts/tests/Common.Tests.ps1

```diff
@@ -0,0 +1,13 @@
+Describe "Haskell" {
+    It "GHC" {
+        "ghc --version" | Should -ReturnZeroExitCode
```

Comment:
We install 3 versions of GHC, can we also validate that exactly 3 versions are installed?

Reply:
Done